scripts/Install-VSCode.ps1

```diff
@@ -100,19 +117,28 @@ param(
 )
```

Comment:
What if you added an `Architecture` parameter that defaults to `64bit` but also allows a `32bit` value.  Then you could test the specific install path (either under Program Files or Program Files (x86) and download the specified bit-version to install.

Reply:
Yep - easy inclusion. I'll add it now.